File: 2581.py
# 함수부
def prime(n):
    if n == 1 : return False
    elif n == 2 : return True
    else:
        for i in range(2, n):
            if n % i == 0 : return False
        return True

# 입력부와 변수선언
m = int(input())
n = int(input())
res = [] # 소수를 담을 리스트

# 연산
for i in range(m, n + 1):
    if prime(i) : res.append(i)

# 출력
if len(res) == 0:
    print(-1)
else:
    print(sum(res))
    print(min(res))

File: 2581.cpp
#include <iostream>
using namespace std;

int f(int n) {
	if(n == 1) return 0;
	
	else if(n == 2) return 1;
	
	else {
		for(int i = 2; i < n; i++) {
			if(n%i==0) return 0;
		}
		return 1;
	}
}

int main(void) {
	int M, N, sum = 0, min = 0;
	cin >> M >> N;
	
	for(int i = M; i <= N; i++) {
		if(f(i) == 1) {
			sum += i;
			
			if(min == 0) min = i;
		}
	}
	
	if(min == 0)
		cout << -1;
	
	else {
		cout << sum << endl;
		cout << min;
	}
	
	return 0;
}
